src/js/components/Box/README.md

```diff
@@ -760,6 +760,12 @@ string
     xlarge
     string
 }
+{
+  corners: [[object Object]
+[object Object]
+[object Object]
```

Comment:
Readme files should only include a link to the documentation.

Reply:
Ok, I'll review. Thanks for the feedback.